bundle.go

```diff
@@ -151,16 +146,6 @@ func (b Bundle) IsPatchUpgrade(other Bundle) (bool, error) {
 }
 
 func (b Bundle) Validate() error {
-	if len(b.Changelogs) == 0 {
```

Comment:
This mostly now allows us to slim down the operators which still have some dummy changelog.